src/problem/func/func_SLIM_VMAG.c

```diff
@@ -216,11 +216,11 @@ void FUNC_SLIM_VMAG_eval_step(Func* f, Branch* br, int t, Vec* var_values) {
 
     if (!bus_counted[bus_index_t[k]]) {
 
-      dv = BUS_get_v_max(bus)-BUS_get_v_min(bus);
+      dv = BUS_get_v_max_reg(bus)-BUS_get_v_min_reg(bus);
       if (dv < FUNC_SLIM_VMAG_PARAM)
 	dv = FUNC_SLIM_VMAG_PARAM;
 
-      vmid = 0.5*(BUS_get_v_max(bus)+BUS_get_v_min(bus));
+      vmid = 0.5*(BUS_get_v_max_reg(bus)+BUS_get_v_min_reg(bus));
```

Comment:
off topic, but I think it would help to have more description in the header documentation to what each constr and func file implements. It isn't directly clear to me what SLIM_VMAG implements.